### DISCUSSION TITLE
Profilerende bildebruk - Pattern

### DISCUSSION THREAD

**Discussion by gbrownlee** [fremtind/jokul#2353] [Annet]
Vi mangler en definisjon på vår måte å bruke bilder i profilerende og dekorativt sammenheng på digitale løsninger.
Man ser ofte en hvit boks delvis over bildet, men dette viser seg å ha noen utfordringer (https://jokul.fremtind.no/profil/bildebruk).

Dette bør dokumenteres i portalen og på Figma. Kanskje en React-komponent også?

**Comment by Steinop** [comment]
Påbegynt tekst vedr animasjon - bare å hacke videre og komme med innspill. Den er foreløpig veldig ustrukturert...
https://www.figma.com/file/OBJ8HrkvW5FysoDWynP418/Skisser-p%C3%A5-organsimer?node-id=1282%3A8711

**Comment by Slaine2** [comment]
Forslag til struktur og bilder. @MonaHolm72 Fortell meg hva du synes om det, så kan jeg justere det.

https://www.figma.com/file/WsUtKI8PAYTFrZiv6ButZM/Bilder?node-id=7%3A2

**Comment by Slaine2** [comment]
Jeg så også etter bilder for bevegelseseksempler. Fremtind-pinnen skal gå i denne retningen. 

https://www.gettyimages.de/collaboration/boards/IFCgC4UrN0CdcCPnuLkM0g?invitation_code=VPHwyTxF9UCViwxOSTVpEg
